python/python/glide/async_commands/transaction.py

```diff
@@ -2047,6 +2049,60 @@ def zdiffstore(
             RequestType.ZDiffStore, [destination, str(len(keys))] + keys
         )
 
+    def zinterstore(
+        self: TTransaction,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, float]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> TTransaction:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
+        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+        When in cluster mode,`destination` and all keys in `keys` must map to the same hash slot.
+
+        See https://valkey.io/commands/zinterstore/ for more details.
+
+        Args:
+            destination (str): The key of the destination sorted set.
+            keys (Union[List[str], List[Tuple[str, float]]]):The keys of the sorted sets with possible formats:
+                List[str] - for keys only.
+                List[Tuple[str, float]]] - for weighted keys with score multipliers.
+            aggregation_type (Optional[AggregationType]): Specifies the aggregation strategy to apply
+                when combining the scores of elements. See `AggregationType`.
+
+        Command response:
+            int: The number of elements in the resulting sorted set stored at `destination`.
+        """
+        args = _create_z_cmd_store_args(destination, keys, aggregation_type)
+        return self.append_command(RequestType.ZInterStore, args)
+
+    def zunionstore(
+        self: TTransaction,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, float]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> TTransaction:
+        """
+        Computes the union of sorted sets given by the specified `keys` and stores the result in `destination`.
+        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+        When in cluster mode, `destination` and all keys in `keys` must map to the same hash slot.
+
```

Comment:
```suggestion
        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
 
        When in cluster mode, `destination` and all keys in `keys` must map to the same hash slot.

```